CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (middleware) [\#1022](https://github.com/cosmos/ibc-go/pull/1022) Add `GetUnwrappedChannelVersion` to the ICS4Wrapper interface. This function should be used by IBC applications to obtain their own version since the version set in the channel structure may be wrapped many times but middleware. 
```

Comment:
```suggestion
* (middleware) [\#1022](https://github.com/cosmos/ibc-go/pull/1022) Add `GetUnwrappedChannelVersion` to the ICS4Wrapper interface. This function should be used by IBC applications to obtain their own version since the version set in the channel structure may be wrapped many times by middleware. 
```